components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -118,6 +124,27 @@ class InvitationModalMembersStep extends React.PureComponent {
         this.props.onSubmit(users, emails, this.state.usersInputValue);
     }
 
+    shouldShowPickerError = () => {
+        const state = getState();
+
+        const userLimit = getConfig(state).ExperimentalCloudUserLimit;
```

Comment:
This needs to be passed in as a prop to the component in order to ensure the component re-renders if the config were to change. You'll want to wrap this component in a redux connector and pass your config item down through there.